back/app/models/notifications/project_phase_started.rb

```diff
@@ -72,6 +72,7 @@ class ProjectPhaseStarted < Notification
 
     def self.make_notifications_on(activity)
       phase = activity.item
+      return [] unless phase.project.published?
```

Comment:
Could have just used `unless phase.project.hidden?` but felt there was actually no need to notify on these unless the project was published - draft or archived has no real use